config/locales/en.yml

```diff
@@ -376,6 +375,10 @@ en:
       confirmation: Are you sure you want to disconnect your account with Uphold?
       deny: Cancel
       confirm: Disconnect
+    confirm_default_currency_modal:
+      headline: Confirm your preferred deposit currency
+      leadin: You will receive monthly contributions from your Brave fans in this currency in your Uphold account.
```

Comment:
The wording is slightly awkward "from your Brave fans in this currency in your Uphold account".

Reply:
cc @jenn-rhim @mandar-brave